src/OgAccess.php

```diff
@@ -112,18 +111,25 @@ public static function userAccess(EntityInterface $group, $operation, AccountInt
     // To reduce the number of SQL queries, we cache the user's permissions
     // in a static variable.
     if (!$pre_alter_cache) {
-      $permissions = array();
+      $permissions = [];
+      $user_is_group_admin = FALSE;
 
       foreach (Og::getUserMemberships($user) as $membership) {
-
         foreach ($membership->getRoles() as $role) {
 
           /** @var $role RoleInterface */
-          $permissions = array_unique(array_merge($permissions, $role->getPermissions()));
```

Comment:
I think you need to keep the `array_unique`, because you might have the same permissions defined in different roles.